### PR TITLE
qi: test added

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

File: Library/Formula/qi.rb [hunk 1]
```diff
@@ -1,35 +1,39 @@
...
+
+  option "with-sbcl", "Use SBCL instead of CLISP"
 
-  option 'SBCL', 'Use SBCL instead of CLISP'
+  deprecated_option "SBCL" => "with-sbcl"
 
-  if build.include? 'SBCL'
-    depends_on 'sbcl'
+  if build.with? "sbcl"
+    depends_on "sbcl"
```
Comment (mikemcquaid) [inline Library/Formula/qi.rb]: `depends_on "sbcl" => :recommended`

Comment (mikemcquaid) [inline Library/Formula/qi.rb]: er, `:optional`

Comment (bfontaine) [inline Library/Formula/qi.rb]: Right thanks, fixed.

**Comment by mikemcquaid** [comment]
@BrewTestBot test this please

**Pull request closed without merging**